ms1searchpy/main.py

```diff
@@ -1496,6 +1699,18 @@ def get_cat_model_final_pfms(df, hyperparameters, feature_columns):
 
     df1['qpreds'] = df1['seqs'].apply(lambda x: qdict.get(x, 11))
 
+    # df1un = df1u[df1u['qpreds'] <= qval_ok].copy()
+    # df1un['qpreds'] = pd.qcut(df1un['preds'], 10, labels=range(10))
+
+    # qdict = df1un.set_index('seqs').to_dict()['qpreds']
+
+
+    # df1['qpreds'] = df1['seqs'].apply(lambda x: qdict.get(x, 11))
+
+    # # df1['qz'] = df1['seqs'].apply(lambda x: qval_dict.get(qdict.get(x, 11), 1.0))
+
+#to here
+
```

Comment:
There was a merge conflict here during rebasing to the latest code in the master branch. Please, check that I did not break anything.